liTest run scala.scalanative.junit.AssumeInAfter started
liTest scala.scalanative.junit.AssumeInAfter.test started
lwTest assumption in test scala.scalanative.junit.AssumeInAfter.test failed: This assume should not pass, took <TIME>
e3scala.scalanative.junit.AssumeInAfter.test::org.junit.AssumptionViolatedException: This assume should not pass::true
ldTest scala.scalanative.junit.AssumeInAfter.test finished, took <TIME>
e0scala.scalanative.junit.AssumeInAfter.test::::true
liTest run scala.scalanative.junit.AssumeInAfter finished: 0 failed, 0 ignored, 1 total, <TIME>
d
